ROLES.md

```diff
@@ -5,19 +5,16 @@ This document describes the set of roles individuals may have within the Istio c
 * [Role summary](#role-summary)
 * [Collaborator](#collaborator)
 * [Member](#member)
-* [Approver](#approver)
+* [Maintainer](#maintainer)
```

Comment:
We have more approvers than what is listed in  MAINTAINERS.md.  I'm trying to be cautious here so folks don't feel they are left out.

Reply:
Correct. I commented in the PR that I initially just took the WG leads and listed as maintainers, but we need to go through the various OWNERS/CODEOWNERS files and find all maintainers. I guess I'll do that in this PR.